chatbot.py

```diff
@@ -105,18 +106,18 @@
 w3 =  tf.Variable(tf.random_normal([hidden_nodes2,output_classes]))
 b3 = tf.Variable(tf.random_normal([output_classes]))
 y = tf.nn.softmax(tf.matmul(l2_out,w3) + b3)
+y_max = tf.argmax(y,1)
 
 cross_entropy = tf.reduce_mean(-tf.reduce_sum(Y * tf.log(y), reduction_indices=[1]))
 train_step = tf.train.GradientDescentOptimizer(0.005).minimize(cross_entropy)
 
 sess = tf.InteractiveSession()
 tf.global_variables_initializer().run()
 epochs = 2000
-
 for i in range(epochs):
     # print train_x[10],train_y[0]
-    print sess.run(train_step, feed_dict={ X:train_x, Y:train_y })
-    # print sess.run(y)
```

Comment:
this didn't work because it needs feed_dict={X:train_x},
because y depends on X for computations in graph